### DISCUSSION TITLE
get  typeScript error when is set defaultValues to useForm

### DISCUSSION THREAD

**Discussion by angelilton** [react-hook-form/react-hook-form#8145] [Q&A]
hey everyone. I try to set default values  em useForm, but I get this error :

`Type 'Control<{ safra: string; ariaPropriaValue: string; ariaArrendadaValue: string; gastosAdmValue: string; gastosFamiliaValue: string; gastosArrendamentoValue: string; culturas: never[]; }, object>' is not assignable to type 'Control<FieldValues, object>'.ts(2322)
[inputText.tsx(6, 3): ]()The expected type comes from property 'control' which is declared here on type 'IntrinsicAttributes & FormInputProps'`

my controller  filed type:
`type FormInputProps = {
  name: string
  control: Control<FieldValues, object>
  label: string
  isShrink?: boolean
} & TextFieldProps
`
the stranger is if I remove the defaultValues on useForm is all ok 

![image](https://user-images.githubusercontent.com/16200828/162037246-e4e03415-94f0-47b9-98bc-8920ad505ce5.png)

![image](https://user-images.githubusercontent.com/16200828/162037336-9768d7b4-2ea5-49a4-9d7d-f709703ef110.png)

**Comment by Moshyfawn** [comment]
It's very hard to say what's going on provided you've given us only screenshots of your code. Please, consider creating a minimal reproduction codesandbox  instead. 

One thing I can mention is you gotta provide your `FieldValues` type to `useForm`
```ts
useForm<FormValues>()
```
